app-drawer-layout/app-drawer-layout.html

```diff
@@ -138,6 +138,14 @@
           value: '640px'
         },
 
+        /**
+         * True if the app-drawer-layout should fit the size of its container.
+         */
+        fullbleed: {
+          type: Boolean,
+          reflectToAttribute: true
+        },
+
```

Comment:
Let's just add a paragraph to the docs instead of creating a property just for documentation.  This is more of an issue in the docs system that it doesn't provide a good way to document attributes/classes.